test/correctness/document-correctness.py

```diff
@@ -296,14 +297,18 @@ def _run_operation_(op1, op2):
         try:
             func1(*args1, **kwargs1)
         except pymongo.errors.OperationFailure as e:
+            print "Failed func1 with " + str(e)
             exceptionOne = e
         except MongoModelException as e:
+            print "Failed func1 with " + str(e)
             exceptionOne = e
         try:
-            func1(*args2, **kwargs2)
+            func2(*args2, **kwargs2)
```

Comment:
It was a luck/unfortunate that when using `transactional_shim`, `func1` and `func2` are always the same LOL. My bad anyways.